math/src/fft/tests.rs

```diff
@@ -18,42 +18,46 @@ use utils::collections::Vec;
 fn fft_in_place() {
     // degree 3
     let n = 4;
-    let mut p = rand_vector(n);
+    let mut binding = rand_vector(n);
+    let p = binding.as_mut_slice();
```

Comment:
why do we need to do it like this? Wouldn't doing `polynom::eval_many(p.as_mut_slice(), &domain);` not work?

Reply:
It will work. I didn't think of it then. Thanks!